cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -919,8 +920,14 @@ export function getJobAsync(tid: number, jid: number, initialFrame: number, init
                 throw new Error(`Task ${tid} doesn't contain the job ${jid}`);
             }
 
-            if (!task.labels.length && task.projectId) {
-                throw new Error(`Project ${task.projectId} does not contain any label`);
+            if (!task.labels.length) {
+                notification.warning({
+                    message: 'No labels',
+                    description: `${task.projectId ? 'Project' : 'Task'} ${
+                        task.projectId || task.id
+                    } does not contain any label`,
+                    placement: 'topRight',
```

Comment:
Can we add a link to a page where labels could be added?

Reply:
Moved to component and added link to project\task